helix-control-m/2-external-monitoring-tools-examples/alerts-to-helix-itsm/README.md

```diff
@@ -86,3 +87,20 @@ BMC Helix Remedy ITSM or on-prem with REST APIs enabled.
   * will compose a URL for the case to display, if the alert is that of a job.
   * There is a feature for those that want to send an email.
     * If you want to send an email and not the ticket, you will need to comment all lines that send the ticket and ensure variables set at the return are not used in messages later.
+
+## How to use it
+
+### Installation
+
+* Clone this repository
+
+* Install the Automation API CLI in the system you will use to receive the alerts and process them to forward to Helix ITSM.
+
+### Configuration
+
+* Modify your tktvars.json with the appropriate information for your environment
+  * You will need credentials and other information from both Helix Control-M and Helix ITSM
+
+* Make your Helix Control-M necessary configurations using the CTM CLI or the scripts in the resources directory
```

Comment:
Again, list the necessary configurations here. It might be redundant but it makes this readme a single source of truth.

Reply:
I expect that the code and files should be the single source of truth. That is why the json files have comments and the code is, in my opinion, appropriately commented. Please send me a guidelines doc of what you want the readme to be, and we can look at modifying this and all others to comply in future iterations.